webapp/src/Controller/PublicController.php

```diff
@@ -133,7 +133,16 @@ public function scoreboardDataZipAction(RequestStack $requestStack, string $proj
 
         $publicPath = realpath(sprintf('%s/public/', $projectDir));
         foreach ($assetMatches[1] as $file) {
-            $zip->addFile($publicPath . '/' . $file, $file);
+            $filepath = realpath($publicPath . '/' . $file);
+            if (
+                substr($filepath, 0, strlen($publicPath)) !== $publicPath &&
+                substr($filepath, 0, strlen($vendorDir)) !== $vendorDir
+            ) {
+                // Path outside of known good dirs: path traversal
```

Comment:
Do we want to write something to a log or even throw an internal error?

Reply:
For now I'm just skipping it, not knowing whatever symlinks or other strange things people may be using... just nog exporting them in the zip

Reply:
I don't think we currently log things from the webinterface. Of course we could but at least currently there seems to be no expectation of logging from web UI interactions.